src/DocBlock/Tags/Return_.php

```diff
@@ -69,4 +69,29 @@ public function __toString()
     {
         return $this->type . ' ' . $this->description;
     }
+
+    private static function splitBodyIntoTypeAndTheRest(string $body) : array
+    {
+        $type = '';
+        $nestingLevel = 0;
```

Comment:
We will need to support multi byte chars I think

Reply:
:thinking: Shite. Forgot :man_facepalming:

Reply:
There; amended the original test to include emoji's (which are multibyte characters)

Reply:
I would expect the usage of `mb_strlen` here. since the current string length will be incorrect when using multi byte characters. This will cause issues when you have a type only return like `@return My😁Class` I think in this case a part of the class name will be seen as description.

Also make sure that  ` trim(substr($body, strlen($type)));` when dealing with multi byte characters. 
will show you what i mean. https://3v4l.org/NBEj7

Reply:
Added extra tests to ensure Emoji's /Multibyte characters won't cause issues